src/Jackett.Common/Definitions/nyaasi.yml

```diff
@@ -166,17 +164,33 @@ search:
       text: "{{ or (.Result.title_spanish) (.Result.title_default) }}"
       filters:
         - name: re_replace
-          args: ["(?i)\\b(\\d{2,})(st|nd|rd|th) Season\\b", "$0 S$1"]
+          args: ["(?i)\\b((?:S|Seasons?|EP?|Episodes?)\\s?)(\\d+)(?:\\-|[\\s~\\+àa&]+)(\\d+)\\b", "$1$2-$3"]
         - name: re_replace
-          args: ["(?i)\\b(\\d)(st|nd|rd|th) Season\\b", "$0 S0$1"]
+          args: ["(?i)\\b(?:S|Seasons?)\\s?(\\d+(?:-\\d+)?)[\\s\\-]+(?:EP|Episodes?)\\s?(\\d+(?:-\\d+)?)\\b", "$0 S$1E$2"]
         - name: re_replace
-          args: ["(?i)\\bSeason (\\d{2,})\\b", "$0 S$1"]
+          args: ["(?i)\\b(?:S|Seasons?)\\s?(\\d+(?:-\\d+)?)(?:.+?)(?:EP|Episodes?)\\s?(\\d+(?:-\\d+)?)\\b(?!(?:-\\d+)?\\sS\\d+(?:-\\d+)?E\\d+(?:-\\d+)?)", "$0 S$1E$2"]
         - name: re_replace
-          args: ["(?i)\\bSeason (\\d)\\b", "$0 S0$1"]
+          args: ["\\b (II) - (\\d+)[\\s\\-~\\+àa&]+(\\d+)", " $1 S02 - $2-$3"]
         - name: re_replace
           args: ["\\b (II) - (\\d+)", " $1 S02 - $2"]
+        - name: re_replace
+          args: ["\\b (III) - (\\d+)[\\s\\-~\\+àa&]+(\\d+)", " $1 S03 - $2-$3"]
         - name: re_replace
           args: ["\\b (III) - (\\d+)", " $1 S03 - $2"]
+        - name: re_replace
+          args: ["(?i)\\b(\\d+)(st|nd|rd|th) Season\\b", "$0 S$1"]
+        - name: re_replace
+          args: ["(?i)\\b(S\\d+(?:-\\d+)?) - (\\d+)[\\s\\-~\\+àa&]+(\\d+)\\b", "$1E$2-$3"]
+        - name: re_replace
+          args: ["(?i)\\b(S\\d+(?:-\\d+)?) - (\\d+)\\b", "$1E$2"]
+        - name: re_replace
+          args: ["(?i)\\b(.+? - )(\\d+)[\\s\\-~\\+àa&]+(\\d+)( [\\[\\(])\\b", "$1S01E$2-$3$4"]
+        - name: re_replace
+          args: ["(?i)\\b(.+? - )(\\d+)( [\\[\\(])\\b", "$1S01E$2$3"]
```

Comment:
Saw a trend of `Title - 01 [1080p]` or `Title - 17-24 (WEBRiP)` so using it identify S01 releases. While it's likely to misidentify, I couldn't actually find any cases. Could use `\\d{1,3}` to make that less likely, or just remove altogether.